oz/redis_sessions/middleware.py

```diff
@@ -24,7 +24,11 @@ def _session_key(self):
 
             if not session_id:
                 session_id = oz.redis_sessions.random_hex(20)
-                self.set_secure_cookie("session_id", session_id.encode('utf-8'), httponly=True)
+                self.set_secure_cookie(
+                    name="session_id",
+                    value=session_id.encode('utf-8'),
+                    domain=oz.settings["cookie_domain"],
```

Comment:
Can we do a check to see if `cookie_domain` exists in `oz.settings` and add it to `set_secure_cookie` kwargs only in that case? We don't want to introduce breaking changes.